src/messages/TextMessage.vue

```diff
@@ -131,6 +153,10 @@ export default {
         first.getMonth() === second.getMonth() &&
         first.getDate() === second.getDate()
       return sameDay
+    },
+    scrollToParentMessage(data) {
+        const parentMesage = data.currentTarget.dataset.parent
+        document.getElementById('message' + parentMesage).scrollIntoView()
```

Comment:
eventuell parentMesage prüfen ob vorhanden und danach scrollIntoView() -> ansonsten könnte cannot scrollIntoView of undefined error